drizzlepac/astrodrizzle.py

```diff
@@ -40,7 +40,6 @@
 from stsci.tools import teal, logutil, textutil
 
 from . import adrizzle
-from . import ablot
```

Comment:
@sosey This crashes the code. Why did you remove this import? I am making a PR to restore this one import.